source/octod/core.d

```diff
@@ -175,12 +175,14 @@ struct HTTPConnection
 
         Params:
             url = GitHub API method URL (relative)
+            accept = optional argument with custom `Accept` header to
+                specify for this one request
 
         Returns:
             Json body of the response. If response is multi-page, all pages
             are collected and concatenated into one returned json object.
      **/
-    Json get ( string url )
+    Json get ( string url, string accept = "" )
```

Comment:
> OK, but then there is a difference between null and "" and you explicitly choose to use "" to not care about which case is used. 

D type system and runtime doesn't make any distinction between those two cases. Your interpretation of tristate would be a personal convention, quite a dangerous thing to rely on. For expressing tri-state in idiomatic manner one uses `std.typecons.Nullable!T`.

In D1 difference is more noticable because assigning null to array forces re-allocation instead of stomping. With D2 runtime it is not observable in reliable manner.